E01-Geometry/Dovydas/test/GeometryTest.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace Kilo;
+
+use PHPUnit\Framework\TestCase;
+
+class GeometryTest extends TestCase
+{
+    public function testCanCalculateSquarePerimeterAndArea(): void
+    {
+        $geometry = new Geometry();
+        $shape = new Square(10,10);
```

Comment:
Are `10` and `10` specific for this test. Have you tested any other numbers?
Move these numbers into constants or even better -- make `dataProvider` with a bunch of numbers, so you would test more numbers as instead just 1 set of data.